Cargo.lock

```diff
@@ -1650,13 +2026,22 @@ dependencies = [
  "slab",
 ]
 
+[[package]]
+name = "fxhash"
+version = "0.2.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "c31b6d751ae2c7f11320402d34e41349dd1016f8d5d45e48c4312bc8625af50c"
+dependencies = [
+ "byteorder",
+]
+
 [[package]]
 name = "generic-array"
 version = "0.12.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "ffdf9f34f1447443d37393cc6c2b8313aebddcd96906caf34e54c68d8e57d7bd"
 dependencies = [
- "typenum 1.15.0",
+ "typenum 1.16.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Did you try compiling this with the node? It could be that we need to update our fork of typenum: https://github.com/encointer/typenum.

We should probably do this anyhow, rather sooner than later.

Reply:
Ah wait, we actually use the newest typenum v1.16.0 in substrate fixed already. So everything should be fine.